EIPS/eip-5773.md

```diff
@@ -0,0 +1,362 @@
+---
+eip: 5773
+title: Multi-Resource Token
+description: An interface for Multi-Resource tokens.
+author: Bruno Škvorc (@Swader), Cicada (@CicadaNCR), Steven Pineda (@steven2308), Stevan Bogosavljevic (@stevyhacker), Jan Turk (@ThunderDeliverer)
+discussions-to: https://ethereum-magicians.org/t/multiresource-tokens/11326
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-10-10
+requires: 165, 721
+---
+
+## Abstract
+
+The Multi-Resource NFT standard allows for the construction of a new primitive: context-dependent output of multimedia information per single NFT.
+
+An NFT can have multiple resources (outputs), and orders them by priority. They do not have to match in mimetype or tokenURI, nor do they depend on one another. Resources are not standalone entities, but should be thought of as “namespaced tokenURIs” that can be ordered at will by the NFT owner, but only modified, updated, added, or removed if agreed on by both the owner of the token and the issuer of the token.
+
+## Motivation
+
+In the four years since the original [EIP-721](./eip-721.md) was published, the need for additional utility resulted in countless implementations on how to provide it. The Multi-Resource Non-Fungible Token standard improves upon it in the following areas:
+
+- [Cross-metaverse compatibility](#cross-metaverse-compatibility)
+- [Multi-media output](#multi-media-output)
+- [Media redundancy](#media-redundancy)
+- [NFT evolution](#nft-evolution)
+
+### Cross-metaverse compatibility
+
+Cross-metaverse compatibility could also be referred to as cross-engine compatibility and (for example) solves the issue where cosmetic item for game A is not portable into game B because the engines are different - it is not a simple matter of just having said cosmetic item, or an NFT.
+
+With Multi-Resource NFTs, it is.
+
+One resource is a cosmetic item for game A, an actual cosmetic item file. Another is a cosmetic item file for game B. A third is a generic resource intended to be shown in catalogs, marketplaces, portfolio trackers - a representation, stylized thumbnail, or animated demo or trailer of the cosmetic item that renders outside of any of the two games.
+
+When using the NFT in such a game, not only don't the game developers need to pre-build the asset into the game and then allow it based on NFT balance in the logged in web3 account, but the NFT has everything it needs in its cosmetic item file, making storage and ownership of this cosmetic item decentralized and not reliant on the game development team.
```

Comment:
```suggestion
This EIP adds a layer of abstraction, allowing game developers to directly pull asset data from a user's NFTs instead of hard-coding it.
```